tests/Feature/CreateTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Tests\Feature;
+
+
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Illuminate\Support\Facades\Artisan;
+use Tests\TestCase;
+
+class CreateTest extends TestCase
+{
+    use DatabaseMigrations;
+    protected function setUp(): void
+    {
+        parent::setUp();
+        Artisan::call('passport:install');
```

Comment:
move this into the TestCase class, otherwise you will have to do it in any test file.